vault/external_tests/quotas/quotas_test.go

```diff
@@ -126,6 +127,42 @@ func waitForRemovalOrTimeout(c *api.Client, path string, tick, to time.Duration)
 	}
 }
 
+func TestQuotas_RateLimitQuota_DupName(t *testing.T) {
+	conf, opts := teststorage.ClusterSetup(coreConfig, nil, nil)
+	cluster := vault.NewTestCluster(t, conf, opts)
+	cluster.Start()
+	defer cluster.Cleanup()
+	core := cluster.Cores[0].Core
+	client := cluster.Cores[0].Client
+	vault.TestWaitActive(t, core)
+
+	// create a rate limit quota w/ 'secret' path
+	_, err := client.Logical().Write("sys/quotas/rate-limit/secret-rlq", map[string]interface{}{
+		"rate":  7.7,
+		"burst": 8,
+		"path":  "secret",
+	})
+	require.NoError(t, err)
+
+	s, err := client.Logical().Read("sys/quotas/rate-limit/secret-rlq")
+	require.NoError(t, err)
+	require.NotEmpty(t, s.Data)
+
+	// create a rate limit quota w/ empty path (same name)
+	_, err = client.Logical().Write("sys/quotas/rate-limit/secret-rlq", map[string]interface{}{
+		"rate":  7.7,
+		"burst": 8,
+		"path":  "",
+	})
+	require.NoError(t, err)
```

Comment:
So when I initially dug into this issue, it was my intuition that this should actually error -- i.e. it would seem you should only be able to mutate tunable fields of a resource quota, `path` not being one of them.

So in this example case, there is no longer a rate limit quota at the root level?

Reply:
There was no defined behavior in that case. We were allowing updates to all fields. I guess we can allow updates to the `path` field, which essentially means that we can "move" the quota around. I don't see any problems with this. Do you?

Reply:
Not inherently no, but we should make sure that's documented.

Reply:
Pushed a doc update.